### DISCUSSION TITLE
[Question][Hw4] JSON parser

### DISCUSSION THREAD

**Discussion by john-mai-2605** [prosyslab-classroom/cs348-information-security#570] [Q&A]
Are the hidden bugs already added to the parser? 
That is, do we have to discover all 15 bugs now or we just need to discover 3 public bugs for now, and the hidden bugs will be added and tested by TA later

**Comment by mickowale** [comment]
The criteria mentions there is only three. The bugs are "division by zero" errors you can search for in the source code.

**Comment by RiceBiscuits** [comment]
We will evaluate your fuzzer using a json parser contained 12 more bugs.
In this homework, we will not push the json parser used for the evaluation to student's repositories 
Thank you.

**Comment by john-mai-2605** [comment]
Got it. Thank you. I will leave this open since other students may have the same question

**Comment by john-mai-2605** [comment]
Closed
